### PR TITLE
PWGDQ: KF Vertexing in DileptonHadron task 

### PR DESCRIPTION
**Pull request by torkjellsdatter**
Changes to the PWGDQ DileptonHadron Task: 
- Implemented Run2 definitions of decay length and pseudoproper decay time in VarManager.cxx, VarManager.h and TableReader
- Implemented 3 prong KF vertexing

### CODE REVIEW

**Comment by dsekihat** [comment]
Hello Ida, can you adjust clang-format?

File: PWGDQ/Core/CutsLibrary.cxx [hunk 1]
```diff
@@ -2043,13 +2050,29 @@ AnalysisCut* o2::aod::dqcuts::GetAnalysisCut(const char* cutName)
     return cut;
   }
 
-  if (!nameStr.compare("kaonPIDnsigma")) {
-    cut->AddCut(VarManager::kTPCnSigmaKa, -3.0, 3.0);
+  if (!nameStr.compare("kaonPIDnsigma2")) {
```
Comment (dsekihat) [inline PWGDQ/Core/CutsLibrary.cxx:2053]: You repeat kaonPIDnsigma2 3 times. Is this expected?

Comment (torkjellsdatter) [reply PWGDQ/Core/CutsLibrary.cxx:2052]: Thank you for noticing! Should be corrected now

Comment (dsekihat) [reply PWGDQ/Core/CutsLibrary.cxx:2052]: I am very sorry. It seems you have removed kaonPIDnsigma. Is it on the purpose?

File: PWGDQ/Tasks/tableReader.cxx [hunk 2]
```diff
@@ -991,7 +1002,7 @@ struct AnalysisSameEventPairing {
       if constexpr ((TPairType == pairTypeMuMu) && muonHasCov) {
         // LOGP(info, "mu1 collId = {}, mu2 collId = {}", t1.collisionId(), t2.collisionId());
         dileptonExtraList(t1.globalIndex(), t2.globalIndex(), VarManager::fgValues[VarManager::kVertexingTauz], VarManager::fgValues[VarManager::kVertexingLz], VarManager::fgValues[VarManager::kVertexingLxy]);
-        dileptonInfoList(t1.collisionId(), event.posX(), event.posY(), event.posZ());
+        dileptonInfoList(t1.globalIndex(), event.posX(), event.posY(), event.posZ());
```
Comment (iarsene) [inline PWGDQ/Tasks/tableReader.cxx:1005]: I don't remember this change. Was this something we discussed or it is also an artefact of the merging?

**Comment by iarsene** [comment]
Hi @torkjellsdatter, @dsekihat,
It seems to me that there are a few unwanted changes done in this PR due to the merging of the master into Ida's local clone. Please not merge until this is solved.

**Comment by dsekihat** [comment]
Hi @iarsene @torkjellsdatter , I see. Thank you for detailed check. What about setting this PR as WIP?

**Pull request closed without merging**